endpoints/openrtb2/auction.go

```diff
@@ -1711,9 +1706,24 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 	return resolvedRequest, impExtInfoMap, nil
 }
 
+func setImpExtInfoMap(imp json.RawMessage, impData ImpExtPrebidData, impExtInfoMap map[string]exchange.ImpExtInfo) error {
+	impId, err := jsonparser.GetString(imp, "id")
+	if err != nil {
+		if err.Error() == "Key path not found" {
+			return errors.New("request.imp[0] missing required field: \"id\"\n")
+		}
+		return err
+	}
+	echoVideoAttributes := false
```

Comment:
I noticed that this part of reading and setting `echoVideoAttributes` in the `ImpExtInfo` was only done for imps that had a stored Imp associated with it, in the old code but, now we are setting this for all imps regardless of whether it has a stored imp associated with it or not. Was this something that was discovered missing for imps that didn't have a stored imp associated with it, when you touched this code?

Reply:
You're right that the reading/setting of `echoVideoAttributes` and `StoredImp` only happened for imps that had a stored imp associated with it. The only thing that's needed to be read/set when there isn't a stored request, is the `passthrough` value found from the imp.

So initially I created a new function to handle the setting of these values cause I thought there was enough duplicate code between when there is a stored request vs. when there isn't, but now I think it makes more sense to remove the new function all together, and leave the original code for when there is a stored request, and when there isn't, I'll just read and set the `passthrough` value to the `impExtInfoMap`.